modules/serviceregistry/src/main/resources/OSGI-INF/serviceregistry-endpoint.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Isn't this unnecessary due to the OSGi annotations?

Reply:
I don't really know. Most, if not all, our current files contain this, and it's also part of examples in the official OSGI docs, so I'm gonna leave it in for now.

Reply:
I actually meant this as a rhetorical question. You can use annotations or specify the OSGi components in this XML file which then needs to be referenced in the bundle plug-in configuration in the `pom.xml` where this is missing. This is also why, for example, [the hello world bundle has no HSGi XML files](https://github.com/opencast/opencast/tree/b19b51be54fee1fca70503f2cd886a0a6e87e0a7/modules/hello-world-impl/src). Using both makes no sense and will only end in confusion when the next developer tries to change e.g. bindings in here and it does not work.

Reply:
I just realized I completely misunderstood your comment. Never mind, you're right, gonna remove that.